opentreemap/treemap/templates/treemap/partials/udf_edit_field.html

```diff
@@ -9,6 +9,7 @@
 </select>
 {% elif field.type == 'date' %}
 <input class="form-control" type="text" data-field-name="{{ field.name }}"
+       data-date-start-date="{{ datepicker_start_date|date:request.instance.short_date_format|datepicker_format }}"
```

Comment:
This is a nice, small, working fix to a temporary problem (living trees were planted before 1900, so we need to remove this date restriction from the application in general) however the most common workflow has a problem.

1. On the edit view of a tree detail page, scroll to the planting site stewardship section
2. Focus the date field, enter "8/5/16" then immediately click the "+" button without unfocusing the date field.

This will add a stewardship row with the date 01/01/1900, which is not what the user entered, and there is no delete button to correct it.

I would suggest attempting to add some JS that checks the value and shows some error "toast" rather than adding a row if the date is equal to the minimum value. In this situation, we may want to consider setting the minimum on the date picker to 1899-12-31 so that the legitimate minimum value is still selectable.